# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/targets/angle_gpu_info_util/moz.build
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/targets/angle_gpu_info_util/moz.build
--- a/gfx/angle/targets/angle_gpu_info_util/moz.build	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/targets/angle_gpu_info_util/moz.build	Sat Sep 07 13:38:36 2019 +0000
@@ -2,16 +2,17 @@
 
 include('../../moz.build.common')
 
+DEFINES['ANGLE_ENABLE_D3D11'] = True
+DEFINES['ANGLE_ENABLE_D3D9'] = True
 #DEFINES['ANGLE_IS_64_BIT_CPU'] = True
 #DEFINES['CERT_CHAIN_PARA_HAS_EXTRA_FIELDS'] = True
 #DEFINES['CHROMIUM_BUILD'] = True
 #DEFINES['COMPONENT_BUILD'] = True
 DEFINES['DYNAMIC_ANNOTATIONS_ENABLED'] = '1'
 #DEFINES['FULL_SAFE_BROWSING'] = True
-DEFINES['GPU_INFO_USE_SETUPAPI'] = True
 DEFINES['NOMINMAX'] = True
 #DEFINES['NO_TCMALLOC'] = True
-DEFINES['NTDDI_VERSION'] = '0x0A000003'
+DEFINES['NTDDI_VERSION'] = 'NTDDI_WIN10_RS2'
 #DEFINES['PSAPI_VERSION'] = '2'
 #DEFINES['SAFE_BROWSING_CSD'] = True
 #DEFINES['SAFE_BROWSING_DB_LOCAL'] = True
@@ -45,7 +46,6 @@
 
 #CXXFLAGS += [
 #    '/bigobj',
-#    '/d2FastFail',
 #    '/D__DATE__=',
 #    '/D__TIME__=',
 #    '/D__TIMESTAMP__=',
@@ -103,6 +103,7 @@
 #    '/we4244',
 #    '/we4456',
 #    '/we4458',
+#    '/we4715',
 #    '/we4800',
 #    '/we4838',
 #    '/WX',
@@ -128,6 +129,7 @@
 ]
 
 OS_LIBS += [
+    'dxgi',
     'setupapi',
 ]
 